streaming/java/streaming-api/src/main/java/org/ray/streaming/jobgraph/JobGraph.java

```diff
@@ -0,0 +1,70 @@
+package org.ray.streaming.jobgraph;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Job graph, the logical plan of streaming job.
+ */
+public class JobGraph implements Serializable {
+
+  private static final Logger LOG = LoggerFactory.getLogger(JobGraph.class);
+
+  private final String jobName;
+  private final Map<String, String> jobConfig;
+  private List<JobVertex> jobVertexList;
+  private List<JobEdge> jobEdgeList;
+
+  public JobGraph(String jobName, Map<String, String> jobConfig) {
+    this.jobName = jobName;
+    this.jobConfig = jobConfig;
+    this.jobVertexList = new ArrayList<>();
+    this.jobEdgeList = new ArrayList<>();
+  }
+
+  public void addVertex(JobVertex vertex) {
+    this.jobVertexList.add(vertex);
+  }
+
+  public void addEdge(JobEdge jobEdge) {
+    this.jobEdgeList.add(jobEdge);
+  }
+
+  public List<JobVertex> getJobVertexList() {
+    return jobVertexList;
+  }
+
+  public List<JobEdge> getJobEdgeList() {
+    return jobEdgeList;
+  }
+
+  public String getGraphViz() {
+    return "";
```

Comment:
Why return an empty string? Can we return a string which can be visualized by `graphviz`?

Reply:
This used to be:
![image](https://user-images.githubusercontent.com/23303886/71970282-55afbe80-3243-11ea-90ca-de31b61cc404.png)

> Can we return a digraph string which can be visualized by `graphviz`?
Will do.